Jellyfin.Api/Controllers/DashboardController.cs

```diff
@@ -211,9 +214,8 @@ public ActionResult GetWebClientResource([FromRoute] string resourceName)
             {
                 return Redirect("index.html?start=wizard#!/wizardstart.html");
             }
-
-            var stream = new FileStream(_resourceFileManager.GetResourcePath(basePath, path), FileMode.Open, FileAccess.Read);
-            return File(stream, MimeTypes.GetMimeType(path));
+            
```

Comment:
Getting build errors on this line
`  DashboardController.cs(217, 1): [SA1028] Code should not contain trailing whitespace`